assets/common/js/models/GameController.js

```diff
@@ -45,13 +45,20 @@ GameController.prototype.stop = function() {
  * Validate the current value of userInputText.
  */
 GameController.prototype.enterWord = function() {
-
+    if (this.words.includes(this.userInputText)) ; // Word typed correctly
```

Comment:
Just a reminder here that this.words is an array of Word objects, not a string variable. But userInputText is a string variable. So Array.includes() function is not going to work the way we expect here. Need to do a bit more for correct comparison. Thanks!